packages/orbit-components/.storybook/orbitDecorator.tsx

```diff
@@ -16,6 +19,16 @@ const OrbitDecorator = (storyFn, context) => {
     };
   }, []);
 
+  const handleCopyCode = codeToCopy => {
+    navigator.clipboard.writeText(codeToCopy);
+
+    setIsCopied(true);
+
+    setTimeout(() => {
+      setIsCopied(false);
+    }, 2000);
```

Comment:
If I understand correctly, you set the timeout value to have a timeframe between showing to user a "tooltip" (or any other form of showing a statement) that the code was copied (?)
@domihustinova

Reply:
Yes, to inform user that the code was copied for a given time before the original text is displayed again.